### ISSUE TITLE
@cubejs-client/ngx : error when running project with Angular 6.1.10

### ISSUE THREAD

**Issue by zegravity** [cube-js/cube.js#204]
**Describe the bug**
I followed this blog post https://statsbot.co/blog/angular-dashboard-tutorial-with-cubejs/ to install CubeJs Client in my Angular project. I'am using Angular 6.1.10.

After installing all those packages ng2-charts @cubejs-client/core @cubejs-client/ngx, I have the following error when I launch my project : 

```
WARNING in ./node_modules/@cubejs-client/ngx/dist/cubejs-client-ngx.umd.js
14105:1017-1100 Critical dependency: the request of a dependency is an expression

WARNING in ./node_modules/@cubejs-client/ngx/dist/cubejs-client-ngx.umd.js
14105:551-572 Critical dependency: the request of a dependency is an expression

WARNING in ./node_modules/@cubejs-client/ngx/dist/cubejs-client-ngx.umd.js
System.import() is deprecated and will be removed soon. Use import() instead.
For more info visit https://webpack.js.org/guides/code-splitting/

WARNING in ./node_modules/@cubejs-client/ngx/dist/cubejs-client-ngx.umd.js
System.import() is deprecated and will be removed soon. Use import() instead.
For more info visit https://webpack.js.org/guides/code-splitting/
```

For your information when I click on the codesandbox demo, it displayed the same error  ( https://codesandbox.io/s/922rqkr4nr )

Thanks in advance for your help.

BR

**Comment by paveltiunov** [comment]
Hey @zegravity ! I believe it's due to it references umd build by mistake. Just shipped fix in v0.10.37. Could you please update and check?

Thanks for heads up regarding code sandbox link. It seems broken but due to some backward incompatible changes were introduced in this sandbox dependencies. We'll upgrade versions and update link to code sandbox: https://codesandbox.io/s/cubejs-client-ngx-demo-cs1kg?fontsize=14

**Comment by zegravity** [comment]
Hi @paveltiunov 
Thanks for your quick response. It is working fine now !

Have a nice day.
